tests/test_trade_pipeline.py

```diff
@@ -1012,8 +1012,8 @@ def test_withdraw_reward_after_unilateral_cancel(self):
         invoice = add_invoice("robot", response.json()["earned_rewards"])
         signed_payout_invoice = sign_message(
             invoice,
-            passphrase_path=f"tests/robots/{trade.taker_index}/token",
-            private_key_path=f"tests/robots/{trade.taker_index}/enc_priv_key",
+            passphrase_path=f"tests/robots/{trade.maker_index}/token",
+            private_key_path=f"tests/robots/{trade.maker_index}/enc_priv_key",
```

Comment:
Well, technically both robots could have a reward to claim. But the robot that cancelled unilaterally in this test is the maker robot (`trade.cancel_order(trade.maker_index)`) and the signed payout invoice is submitted by the taker two lines below `self.client.post(path, body, **taker_headers)` so this change here should not work :dizzy_face: :smile:

Reply:
I will try to research this one once I get a bit og time